packages/eslint-plugin/rules/jsdoc-third-person.js

```diff
@@ -51,7 +51,7 @@ module.exports = iterateJsdoc(
 		// Verbs can include dashes or in some cases also parentheses.
 		if (
 			jsdoc.description &&
-			! jsdoc.description.match( /^[\w\(\)\-]+s\W.*/g )
+			! jsdoc.description.match( /^[\w\(\)\-]+s\W.*/g ) // eslint-disable-line
```

Comment:
```suggestion
			! jsdoc.description.match( /^[\w\(\)\-]+s\W.*/g ) // eslint-disable-line
```

Why is this line being disabled? As mentioned in this comment: https://github.com/google/site-kit-wp/pull/6295#discussion_r1044676281, we should be specific about rules we disable, and why.

I think here we should be able to convert this to a `new RegExp` format, right?